draft-ietf-acme-acme.md

```diff
@@ -1916,8 +1924,14 @@ It MUST NOT contain any characters outside the URL-safe Base64 alphabet and MUST
 NOT contain any padding characters ("=").
 
 ~~~~~~~~~~
+GET /acme/authz/1234/0 HTTP/1.1
```

Comment:
Including all the HTTP and JWS glue in here is counter-productive.  The point of these sections is to highlight the structure of the object you need to send, not the HTTP/JWS mechanics.  The HTTP/JWS stuff obscures that message.